www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -57,24 +44,24 @@ Once you've cloned core-plans locally, go to `core-plans/nginx` and enter the Ha
-* Export the hab artifact to a Helm chart: `hab pkg export helm results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
+* Initialize the Helm client-side: `helm init -c`
+* Export the hab artifact to a Helm chart: `hab pkg export helm --download-deps results/<yourfilename>.hart` This command will look very similar to `hab pkg export helm --download-deps results/tdrew-nginx-1.11.10-20180221053714-x86_64-linux.hart`
 * Exit the Habitat Studio: `exit`
 
 You now have a Helm chart for core-plans/nginx in your core-plans/nginx/nginx-<version>-<datestamp>/ directory
 I suggest you pwd for the above ^^ so you can use it in another tab when you have started your cluster and want to know where your helm chart lives
 
 ### Start a cluster and deploy your helm chart!
 
-Here we are going to set up your minikube, and then deploy our helm chart onto that cluster. So, go back to your `habitat-operator` directory where you left your minikube cluster running and: 
+Here we are going to set up your minikube, and then deploy our helm chart onto that cluster.
 
-* `kubectl create -f examples/rbac` This gives the Habitat Operator the permissions it needs on your RBAC-enabled cluster
-* `kubectl apply -f examples/rbac/habitat-operator.yml` Deploys the Habitat Operator in your minikube cluster 
+* `kubectl apply -f https://raw.githubusercontent.com/kinvolk/habitat-operator/v0.5.1/examples/rbac/habitat-operator.yml` This gives the Habitat Operator the permissions it needs on your RBAC-enabled cluster
+* `kubectl apply -f https://raw.githubusercontent.com/kinvolk/habitat-operator/v0.5.1/examples/rbac/rbac.yml` Deploys the Habitat Operator in your minikube cluster
 * `kubectl -n kube-system create sa tiller` Creates a service account for Tiller in the kube-system namespace
 * `kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller` Creates a ClusterRoleBinding for Tiller
 * `helm init --service-account tiller` Installs Tiller, specifying new service account 
 * `helm version` Check and make sure you have the right version of helm running on your client & server side, (as of printing, v 2.8.1) 
 * `kubectl -n kube-system describe deploy/tiller-deploy` Check and see your service account has been correctly set up per instructions
-* `helm dependency build <what your pwd was from building your Helm + Habitat package>` Reads your new nginx-<version>-<timestamp>/requirements.yaml file and sees the dependency on the habitat-operator. Pulls down the chart that describes the Habitat Operator and embeds it in your application’s chart 
 * `helm install /Users/tdrew/core-plans/nginx/results/nginx-1.11.10-20180220023948/` (or whatever your pwd was -- if it's exactly this we need to talk.) Has Helm install your Habitat Helm package on your Minikube cluster
 
 You will now see output similar to this: 
```

Comment:
Please change this to have a status `1/1` pods. :)

Reply:
will do, thanks.